src/Gui/DlgMacroExecuteImp.cpp

```diff
@@ -188,6 +262,14 @@ void DlgMacroExecuteImp::onUserMacroListBoxCurrentItemChanged(QTreeWidgetItem* i
     }
 }
 
+void DlgMacroExecuteImp::onLineEditFindTextChanged(const QString &){

```

Comment:
The linter is yelling here because in general it's recommended to name the variable, and then use the Q_UNUSED macro on it, so it's explicit that you're deliberately ignoring it.

Reply:
I've made this change.  Waiting for the unit tests to finish, and then I'll push the update.  There was another change I noticed, too.  QtDesigner added native=true or something to the file chooser.  I hate it when software does stuff like that behind my back.